src/06_tuples.py

```diff
@@ -17,25 +17,31 @@
 
 import math
 
+
 def dist(a, b):
     """Compute the distance between two x,y points."""
     x0, y0 = a  # Destructuring assignment
     x1, y1 = b
 
     return math.sqrt((x1 - x0)**2 + (y1 - y0)**2)
 
+
 a = (2, 7)   # <-- x,y coordinates stored in tuples
 b = (-14, 72)
 
 # Prints "Distance is 66.94"
 print("Distance is: {:.2f}".format(dist(a, b)))
 
 
-
 # Write a function `print_tuple` that prints all the values in a tuple
 
 # YOUR CODE HERE
 
+def print_tuple(tup):
+    for num in tup:
+        print(tup)
+
+
 t = (1, 2, 5, 7, 99)
 print_tuple(t)  # Prints 1 2 5 7 99, one per line
 
```

Comment:
@RodBromberg: Don't forget that you'll need the trailing comma after the "1" in the value for your "u" variable (line 49) because of the weird syntax rule dealing with tuples with only a single item in them! 👍